ingest/ingest_record.go

```diff
@@ -22,14 +22,16 @@ import (
 	ingestpb "github.com/indykite/indykite-sdk-go/gen/indykite/ingest/v1beta3"
 )
 
+// IngestRecord function is still required for backward compatibility.
 func (c *Client) IngestRecord(
 	ctx context.Context,
 	record *ingestpb.Record,
 	opts ...grpc.CallOption,
 ) (*ingestpb.IngestRecordResponse, error) {
 	ctx = insertMetadata(ctx, c.xMetadata)
-	resp, err := c.client.IngestRecord(ctx, &ingestpb.IngestRecordRequest{
-		Record: record,
-	}, opts...)
+	resp, err := c.client.IngestRecord( //nolint:staticcheck // backward compatibility.
```

Comment:
Why is the linter comlaining here ?

Reply:
because I'm keeping a deprecated function
But I cannot remove it without the normal process delay: first flag as deprecated then later remove

Reply:
Okay but if I understand correctly here, I would prefer a comment saying something along "IngestRecord is now depreacted and will be removed plus migrate to NEW_METHOD" so that a new des does not start usign this method